static/webpack/webpack.prod.js

```diff
@@ -29,6 +31,14 @@ module.exports = () => {
           }
         }
       ]
-    }
+    },
+    optimization: {
+      minimizer: [
+        new ESBuildMinifyPlugin({
+          target: 'es5',
```

Comment:
would we be able to shrink the bundle just a little bit if we specify specific browser versions? Or maybe, since we're already running babel and importing polyfills, can we just not specify a target?

Reply:
I can check if it's possible to target ie11. I tried not specifying a target and it didn't work since esbuild will introduce some newer javascript during minification if the target is not es5

Reply:
ah dang got it, I guess it makes sense it would undo the babel stuff in order to make the code smaller

Reply:
It is not possible to target ie11. ES5 is the only target that will work for us